### PR TITLE
docs: Add guide pages [JOB-111257]

### PR DESCRIPTION
**Pull request by Aiden-Brine**
### Added

Adds the guide pages to the new docs site

## Testing

Flip through the guide pages using the side nav and the guide landing page to make sure everything looks as expected

Changes can be
[tested via Pre-release](https://github.com/GetJobber/atlantis/actions/workflows/trigger-qa-build.yml)

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: docs/getting-started-with-react/getting-started-with-react.stories.mdx [hunk 1]
```diff
@@ -87,25 +87,7 @@ For this example we are going to build a card that allows you to enter a topic
...
-          </Content>
-        </Card>
-      );
-      async function gifSearch() {
-        setURL(loadingGIF);
-        setURL(await getGIF(topic));
-      }
-    }}
-  </Story>
+  <GifGift />
```
Comment (Aiden-Brine) [inline docs/getting-started-with-react/getting-started-with-react.stories.mdx:90]: By pulling this out into a component in `getGif.tsx` and removing the `Story` we no longer get the unnecessary nesting on Storybook and the functionality will work on the new docs site (it wasn't if I just pulled in this mdx file as is).

Behaviour before on the Storybook site:

Screenshot 2024-12-16 at 1 11 40 PM


Behaviour now:

Screenshot 2024-12-16 at 1 09 34 PM

File: docs/getting-started-with-react/getGIF.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+export async function getGIF(topic: string) {
+  const searchURL = `//api.giphy.com/v1/gifs/random?tag=${topic}&rating=PG&api_key=${gifKey}`;
+  const response = await fetch(searchURL);
+  const json = await response.json();
+
+  return json.data.images.original.url;
+}
+
+export function GifGift() {
```
Comment (Aiden-Brine) [inline docs/getting-started-with-react/getGIF.tsx:20]: See [here](https://github.com/GetJobber/atlantis/pull/2259/files#r1887591570)

File: docs/getting-started-with-react/getting-started-with-react.stories.mdx (hunk 1 above)
Comment (taylorvnoj) [reply docs/getting-started-with-react/getting-started-with-react.stories.mdx:90]: Yesss thank you for doing this! I noticed this last week and wanted to do the same thing. 👍🏻 Everything works still in Storybook and in new docs site

**Review by taylorvnoj — APPROVED** [review]
🚀 Looks great! Ship it!

**Pull request merged**